lib/validator/validator.go

```diff
@@ -34,7 +34,7 @@ var validate *validator.Validate
 var (
 	nameRegex          = regexp.MustCompile("^[a-zA-Z0-9_.-]{1,128}$")
 	interfaceRegex     = regexp.MustCompile("^[a-zA-Z0-9_-]{1,15}$")
-	labelRegex         = regexp.MustCompile("^([a-zA-Z0-9.-]{0,253}/)?[a-zA-Z0-9]([a-zA-Z0-9_.-]{0,61}[a-zA-Z0-9])?$")
+	labelRegex         = regexp.MustCompile(`^([a-zA-Z0-9\_\.\-\/]{0,252}[a-zA-Z0-9])?$`)
```

Comment:
> so it's 252 (alphanumeric+specials) + 1 only alphanumeric so it doesn't end with a special char

The old format was 253 alphanumeric+specials followed by a slash + 61  alphanumric+specials followed by an alphanumeric.  Right?

So, we've gone from total length 253+1+61+1 to 252+1?